deploy/oauth2/deploy-template.yaml

```diff
@@ -415,7 +416,7 @@ spec:
             - name: swagger-api
               mountPath: /api
         - name: cns-manager
-          image: projects.registry.vmware.com/cns_manager/cns-manager:r0.1.1-rc.1
+          image: projects.registry.vmware.com/cns_manager/cns-manager:r0.2.0-rc.6
```

Comment:
What's the plan to change the value to the official release image?

Reply:
We'll change both in `master` and a new `release-0.2` branch when the official image is available, before cutting a new release.